### PR TITLE
fix: Deprecate prettier package, simplify ESLint config

### PR DESCRIPTION
**Pull request by lukelooker**

### CODE REVIEW

**Review by mdodgelooker — APPROVED** [review]
LGTM

File: packages/eslint-config-oss/src/index.js [hunk 1]
```diff
@@ -137,18 +129,14 @@ module.exports = {
         ],
       },
     ],
-    'no-undef': 'off', // Not needed with TS around
     'no-use-before-define': 'off',
-    'prettier/prettier': 'error',
-    'react-hooks/exhaustive-deps': 'warn',
-    'react-hooks/rules-of-hooks': 'error',
```
Comment (lukelooker) [inline packages/eslint-config-oss/src/index.js:144]: Use `'plugin:react-hooks/recommended',` above instead

File: packages/eslint-config-oss/src/index.js [hunk 2]
```diff
@@ -137,18 +129,14 @@ module.exports = {
...
     'react/no-unescaped-entities': 'off',
     'react/prop-types': 'off',
     'sort-keys': 'off',
     'sort-keys-fix/sort-keys-fix': 'error',
+    '@typescript-eslint/no-unused-vars': 'off',
     // Work to enable these soon-ish
     'testing-library/no-node-access': 'off',
-    'testing-library/prefer-screen-queries': 'off',
```
Comment (lukelooker) [inline packages/eslint-config-oss/src/index.js:151]: We pass without trouble 🎉

**Comment by medington** [comment]
Hi Luke, wondering what I should be doing with the failure here:

https://github.com/looker/cloud-services/pull/1031

Sounds like you are getting rid of that package.

**Pull request merged**